jsonnet/components/node-exporter.libsonnet

```
@@ -214,6 +214,21 @@ function(params)
                               '--collector.textfile.directory=' + textfileDir,
                               '--no-collector.btrfs',
                             ],
+                      command: [
+                        '/bin/sh',
+                        '-c',
+                        |||
+                          export GOMAXPROCS=4
+                          # We don't take CPU affinity into account as the container dones't have integer CPU requests.
+                          # In case of error, fallback to the default value.
+                          NUM_CPUS=$(grep -c '^processor' "/proc/cpuinfo" 2>/dev/null || echo "0")
+                          if [ "$NUM_CPUS" -lt "$GOMAXPROCS" ]; then
+                            export GOMAXPROCS="$NUM_CPUS"
+                          fi
+                          echo "Set GOMAXPROCS=$GOMAXPROCS"
```

(suggestion) we can make the logged message look like a regular log message...
```suggestion
                          echo "ts=$(date --iso-8601=seconds) num_cpus=$NUM_CPUS gomaxprocs=$GOMAXPROCS"

```

---

Done.

num_cpus will also ease debugging.

(We get `XXX+00:00` instead of `XXXMSZ`, I don't think this is a problem, it's even better to not make the user thinks it's node_exporter...)

(node_exporter can also `ts=2023-06-19T23:25:13.022Z caller=node_exporter.go:186 level=debug msg="Go MAXPROCS" procs=5` when debug, I don't think this is confusing as there is `Starting node_exporter` in between)